lib/IRGen/GenStruct.cpp

```diff
@@ -40,6 +40,7 @@
 #include "ResilientTypeInfo.h"
 #include "StructMetadataLayout.h"
 
+// TODO: address this missing override warning, don't ignore it.
```

Comment:
Always a reason. I reverted the TODOs. There are no changes to the pragmas in this PR